Makefile

```diff
@@ -158,11 +159,15 @@ smokecheck: clean develop
 	./scripts/avocado run passtest.py
 
 check: clean develop check_cyclical modules_boundaries
-	selftests/checkall
+	AVOCADO_CHECK_LEVEL=0 selftests/checkall
```

Comment:
My point is that by making it explicit you override the system environment which some users (me) could define in `bashrc`. So I'd really prefer having `make check` without overriding the environment variables and only override it for `make check-full`.